<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaMenuPrincipal</class>
 <widget class="QMainWindow" name="ventanaMenuPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1900</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1900</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Men√∫ principal del sistema</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelFondoMenuRealizarComparacion">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1900</width>
      <height>851</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>menuRealizarComparacionFigma.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="labelRealizarEliminacion">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>730</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelVolverInicio">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>20</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelConfiguracion">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelRealizarComparacion">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>450</y>
      <width>591</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableView" name="tableViewVerPerfiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>1261</width>
      <height>671</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIdPerfilComparar">
    <property name="geometry">
     <rect>
      <x>1630</x>
      <y>220</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxUmbralComparacion">
    <property name="geometry">
     <rect>
      <x>1630</x>
      <y>340</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>30.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>35.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>40.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>45.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>55.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>60.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>65.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>70.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>75.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>80.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>85.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>90.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>95.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100.0</string>
     </property>
    </item>
   </widget>
   <zorder>labelFondoMenuRealizarComparacion</zorder>
   <zorder>labelRealizarEliminacion</zorder>
   <zorder>labelVolverInicio</zorder>
   <zorder>labelConfiguracion</zorder>
   <zorder>labelRealizarComparacion</zorder>
   <zorder>lineEditIdPerfilComparar</zorder>
   <zorder>tableViewVerPerfiles</zorder>
   <zorder>comboBoxUmbralComparacion</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
